next.config.js

```diff
@@ -0,0 +1,53 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
```

Comment:
Do we need all this unsplahs things?

Reply:
yes but if we get images to use I will remove it